config.py

```diff
@@ -672,31 +672,37 @@ def __init__(self, paths: OPPaths = None):
         # ==========================================================================================
         # Account Abstraction Configuration
 
-        # === Private Key ===
-
-        self.deployer_key = None
+        self.aa_deployer_key = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
         """
-        Private key to use for deploying 4337 contracts (None by default).
-        Will be used if set, otherwise will prompt users to enter private key.
+        Private key to use for deploying 4337 contracts.
+        Uses the 0th "test junk" mnemonnic key by default.
         """
 
-        self.bundler_key = None
+        self.bundler_key = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Comment:
So the issue with stackup bundler is that it can't recognized private key with `0x`. So we should put this as `self.bundler_key = "ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"`